### DISCUSSION TITLE
Experimental models - NASA GEOS5-FP

### DISCUSSION THREAD

**Discussion by phillipseamore** [open-meteo/open-meteo#705] [Ideas]
I noticed that some experimental AI models are available now in the API. Might I suggest taking a look at another experimental model, GEOS5-FP from NASA. 

https://gmao.gsfc.nasa.gov/GEOS_systems/
https://gmao.gsfc.nasa.gov/GMAO_products/NRT_products.php

Downloads: https://portal.nccs.nasa.gov/datashare/gmao/geos-fp/forecast/Y2024/M03/

The documentation is a bit here and there but it has some interesting features, including 30min steps.

**Comment by patrick-zippenfenig** [comment]
Thanks for the tip! I have not seen this forecast before.

I am not sure if it really offers 30 minutely data. Many files have `xx:30` timestamps, but are still hourly. Also documented in their [docs](https://gmao.gsfc.nasa.gov/pubs/docs/Lucchesi1203.pdf): "Frequency: 1-hourly from 00:30 UTC (time-averaged)".

One exception are `GEOS.fp.fcst.inst_30mn_met_c0720sfc` files which 30 minutes files, but they seems to be limited to Africa with parts of Southern Europe and South America.

![Screenshot 2024-03-21 at 13 52 36](https://github.com/open-meteo/open-meteo/assets/11670456/ecb63c9b-f4c7-4cc9-ab74-de6aeabbd69e)

**Comment by phillipseamore** [comment]
Could the download be truncated? I get global coverage. 

I just made a stack of T2 from midnight to 02 (so 5 steps)  and they are all different ![T2M_in_GEOS fp fcst inst_30mn_met_c0720sfc 202403_5](https://github.com/open-meteo/open-meteo/assets/3772845/92969d39-866f-4730-80b7-4861ea2e73f7)

Constants are here: https://portal.nccs.nasa.gov/datashare/gmao/geos-fp/das/GEOS.fp.asm.const_2d_asm_Nx.00000000_0000.V01.nc4
